external/CMakeLists.txt

```diff
@@ -134,12 +135,23 @@ if (DEFINED EXECUTING_SCRIPT)
         WORKING_DIRECTORY "${REPO_PATH}"
         COMMAND_ERROR_IS_FATAL ANY
       )
+      execute_process(
+        COMMAND "${GIT_EXECUTABLE}" submodule update --init --recursive
+        WORKING_DIRECTORY "${REPO_PATH}"
+        COMMAND_ERROR_IS_FATAL ANY
+      )
     else()
       execute_process(
         COMMAND "${GIT_EXECUTABLE}" clone --depth 1 "${REPO}"
         WORKING_DIRECTORY "${DOWNLOADS_DIR}"
         COMMAND_ERROR_IS_FATAL ANY
       )
+      message("Downloading submodules for ${REPO}")
```

Comment:
Maybe we should print this around line 138 too?

Reply:
I removed this line. It was for debugging.
done.